internal/worker/trace/client.go

```diff
@@ -169,3 +187,59 @@ func newResource(serviceName, serviceID string) *resource.Resource {
 		semconv.ServiceInstanceID(serviceID),
 	)
 }
+
+type tailSamplingProcessor struct {
+	bsp sdktrace.SpanProcessor
+
+	logger logger.Logger
+
+	threshold time.Duration
+}
+
+// OnStart is called when a span is started. It is called synchronously and
```

Comment:
Because the batch span processor has `BlockOnQueueFull: false`?

Reply:
Oh looks like OnStart with a batch span processor is a no-op.

Reply:
Yeah, but I can't guarantee that in the future, so just pass through, even if we know it's currently a no-op.